src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToControlGeneratedAssemblyInfo.cs

```diff
@@ -421,17 +456,17 @@ public void TestPreviewFeatures(bool enablePreviewFeatures, bool generateRequire
 
             if (enablePreviewFeatures && generateRequiresPreviewFeaturesAttribute)
             {
-                if (targetFramework == "net6.0")
-                {
-                    Assert.Equal("Preview", langVersion);
-                    Assert.True(contains);
-                }
-                else
+                if (targetFramework == "net5.0")
                 {
                     // The assembly level attribute is generated only for the latest TFM for the given sdk
                     Assert.False(contains);
                     Assert.NotEqual("Preview", langVersion);
                 }
+                else
```

Comment:
Nit: I think this would be clearer if there were an explicit comparison between `targetFramework` and the latest target framework (`CurrentTFM` or `LatestTargetFramework` or whatever the property ends up being).